Library/Homebrew/missing_formula.rb

```diff
@@ -79,6 +81,9 @@ def blacklisted_reason(name)
           If you wish to use the 2.x release you can install with Homebrew Cask:
             brew cask install ngrok
         EOS
+        when "cargo" then <<~EOS
+          Homebrew provides cargo via: `brew install rust`.
```

Comment:
How about:

```
cargo is part of the rust formula, and can be installed with:
  brew install rust
```

Reply:
I thought about that too, but I wasn't sure about using something so close to the "is _now_ part of..." message, and decided to model the message after the one for pip, which is a nearly identical use case.

That said, if we change this, the pip one should also be adjusted in the same way, right?

Reply:
@waldyrious Yeh, changing `pip` too sounds good! Consistency in general is great 👍

Reply:
Cool, will do.